UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/FssService.cs

```diff
@@ -285,12 +286,12 @@ private CreateBatchRequestModel CreateBatchRequestModel(Batch batchType)
 
             CreateBatchRequestModel createBatchRequest = new()
             {
-                BusinessUnit = "AVCSData",
+                BusinessUnit = _fssApiConfiguration.Value.BusinessUnit,
                 ExpiryDate = DateTime.UtcNow.AddDays(28).ToString("yyyy-MM-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture),
                 Acl = new Acl()
                 {
-                    ReadUsers = new List<string>() { "public" },
-                    ReadGroups = new List<string>() { "public" }
+                    ReadUsers = string.IsNullOrEmpty(_fssApiConfiguration.Value.PosReadUsers) ? new() : _fssApiConfiguration.Value.PosReadUsers.Split(",").ToList(),
```

Comment:
Is there a reason you're checking IsNullOrEmpty on the ReadUsers assignment, but you're not on the ReadGroups assignment?

Reply:
ReadGroups will always have value either public in prod or some group name in pre-prod testing.